src/AbstractMangrove.sol

```diff
@@ -5,12 +5,41 @@ import {MgvLib} from "./MgvLib.sol";
 
 import {MgvOfferMaking} from "./MgvOfferMaking.sol";
 import {MgvOfferTakingWithPermit} from "./MgvOfferTakingWithPermit.sol";
-import {MgvGovernable} from "./MgvGovernable.sol";
+import {MgvAppendix} from "mgv_src/MgvAppendix.sol";
 
-/* `AbstractMangrove` inherits the three contracts that implement generic Mangrove functionality (`MgvGovernable`,`MgvOfferTakingWithPermit` and `MgvOfferMaking`) but does not implement the abstract functions. */
-abstract contract AbstractMangrove is MgvGovernable, MgvOfferTakingWithPermit, MgvOfferMaking {
-  constructor(address governance, uint gasprice, uint gasmax, string memory contractName)
+/* `AbstractMangrove` inherits the three contracts that implement generic Mangrove functionality (`MgvOfferTakingWithPermit` and `MgvOfferMaking`) but does not implement the abstract functions. */
+abstract contract AbstractMangrove is MgvOfferTakingWithPermit, MgvOfferMaking {
+  address immutable appendix;
```

Comment:
appendix <- APPENDIX

Reply:
done in 6a5584b8e81b47b3d87ee08c586416d2cbfed36e